### PR TITLE
Allow enableReorder on DraggableWrap

### PR DESCRIPTION
**Pull request by 89jd**

### CODE REVIEW

**Comment by 89jd** [comment]
Sure I need to be able to toggle it enabled.

To do this, I have added a boolean that you pass to the root view to disable drag completely (basically as a standard flow)

**Comment by diegotori** [comment]
Looks good to me. However, please revert the code formatting changes, since they're polluting the PR. Thanks.

**Comment by 89jd** [comment]
Just noticed now, at work right now, will sort when I am finished :)

File: lib/src/widgets/reorderable_wrap.dart [hunk 1]
```diff
@@ -321,33 +326,33 @@ class _ReorderableWrapState extends State<ReorderableWrap> {
...
-    required this.verticalDirection,
-    required this.minMainAxisCount,
-    required this.maxMainAxisCount,
-    this.header,
-    this.footer,
-    this.controller,
-    this.reorderAnimationDuration = const Duration(milliseconds: 200),
-    this.scrollAnimationDuration = const Duration(milliseconds: 200),
-  });
+  const _ReorderableWrapContent(
```
Comment (diegotori) [inline lib/src/widgets/reorderable_wrap.dart:329]: Fix formatting here.

File: lib/src/widgets/reorderable_wrap.dart [hunk 2]
```diff
@@ -9,9 +9,11 @@ import 'package:flutter/rendering.dart';
 import 'package:reorderables/reorderables.dart';
 
 import './passthrough_overlay.dart';
+
```
Comment (diegotori) [inline lib/src/widgets/reorderable_wrap.dart:12]: Remove un-needed spaces.

**Comment by 89jd** [comment]
done now :)

**Comment by diegotori** [comment]
LGTM.

**Pull request merged**